vhdbuilder/packer/cleanup.sh

```diff
@@ -69,6 +69,7 @@ if [[ "${MODE}" == "windowsVhdMode" && "$SIG_FOR_PRODUCTION" == "True" ]]; then
         az sig image-version show -e $version -i ${SIG_IMAGE_NAME} -r ${SIG_GALLERY_NAME} -g ${AZURE_RESOURCE_GROUP_NAME} | jq .id
         echo "Deleting sig image-version ${version} ${SIG_IMAGE_NAME} from gallery ${SIG_GALLERY_NAME} rg ${AZURE_RESOURCE_GROUP_NAME}"
         az sig image-version delete -e $version -i ${SIG_IMAGE_NAME} -r ${SIG_GALLERY_NAME} -g ${AZURE_RESOURCE_GROUP_NAME}
+        az sig image-version wait --deleted --timeout 1800 -e $version -i ${SIG_IMAGE_NAME} -r ${SIG_GALLERY_NAME} -g ${AZURE_RESOURCE_GROUP_NAME}
```

Comment:
This will increase the build duration. How long will this increase?

Reply:
In my test, the cleanup step only needs several minutes. This step is to improve the successful rate of deleting sig images to reduce the resources that are not cleaned up.

Reply:
Remove this part to save vhd build time.